codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/samples/KDocSamplesGenerator.kt

```diff
@@ -63,10 +76,11 @@ class KDocSamplesGenerator : KotlinIntegration {
             sampleFunctionName(index),
         ).joinToString(separator = ".")
 
-    private fun sampleFunctionName(index: Int): String = "sample${index + 1}"
+    private fun sampleFunctionName(index: Int): String = "sample" + if (index > 0) "$index" else ""
```

Comment:
**Nit**: The second sample is now called `sample1` but I think `sample2` was better (like before).